pkgs/by-name/bl/blender/package.nix

```diff
@@ -201,6 +172,7 @@ stdenv'.mkDerivation (finalAttrs: {
       "-DWITH_OPENIMAGEDENOISE=${if openImageDenoiseSupport then "ON" else "OFF"}"
       "-DWITH_OPENSUBDIV=ON"
       "-DWITH_OPENVDB=ON"
+      "-DWITH_PIPEWIRE=OFF"
```

Comment:
Is there a specific reason, why we want no pipewire?
Or could we make this optional, same as jack?

Reply:
Just saw your not in the commit message, sorry for not looking there first.

Still an option might be nice to have, no?

Reply:
I see, with that It sure makes to wait for upstream defaulting to `ON`.